edocumentation-domain/src/main/java/org/dcsa/edocumentation/domain/validations/AdvanceManifestFilingEBLValidator.java

```diff
@@ -0,0 +1,42 @@
+package org.dcsa.edocumentation.domain.validations;
+
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.dcsa.edocumentation.domain.persistence.entity.AdvanceManifestFilingEBL;
+
+public class AdvanceManifestFilingEBLValidator implements ConstraintValidator<AdvanceManifestFilingEBLValidation, AdvanceManifestFilingEBL> {
+
+  private static final CodeCache VALID_MANIFEST_CODES =
+    CodeCache.of("advancedmanifestfilings.csv", "Advance Manifest Filing Type Code", "Country Code");
+
+  @Override
+  public boolean isValid(AdvanceManifestFilingEBL value, ConstraintValidatorContext context) {
+    if (value == null) {
+      return true;
+    }
```

Comment:
I don't think this is true. *If* this is null - then it would require that all bookings linked to via the `consignmentItems` also have null as `advanceManifestFilings`

Reply:
The value here is an element in the advancemanifestfilings array.